packages/eas-cli/src/graphql/queries/UpdateQuery.ts

```diff
@@ -112,4 +135,142 @@ export const UpdateQuery = {
+                manifestFragment
+                actor {
+                  id
+                  ... on User {
+                    username
+                  }
+                  ... on Robot {
+                    firstName
+                  }
+                }
+                branch {
+                  id
+                  name
+                }
+              }
+            }
+          `,
+          {
+            groupId,
+          },
+          { additionalTypenames: ['Update'] }
+        )
+        .toPromise()
+    );
+  },
+  async viewUpdateGroupsOnBranchAsync({
+    limit,
+    offset,
+    appId,
+    branchName,
+    filter,
+  }: ViewUpdateGroupsOnBranchForAppQueryVariables): Promise<ViewUpdateGroupsOnBranchForAppQuery> {
+    return await withErrorHandlingAsync(
+      graphqlClient
+        .query<ViewUpdateGroupsOnBranchForAppQuery, ViewUpdateGroupsOnBranchForAppQueryVariables>(
+          gql`
+            query ViewUpdateGroupsOnBranchForApp(
+              $appId: String!
+              $branchName: String!
+              $limit: Int!
+              $offset: Int!
+              $filter: UpdatesFilter
+            ) {
+              app {
+                byId(appId: $appId) {
+                  id
+                  updateBranchByName(name: $branchName) {
+                    id
+                    updateGroups(limit: $limit, offset: $offset, filter: $filter) {
+                      id
+                      group
+                      message
+                      createdAt
+                      runtimeVersion
+                      platform
+                      manifestFragment
+                      branch {
+                        id
+                        name
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          `,
+          {
+            appId,
+            limit,
+            offset,
+            branchName,
+            filter,
+          },
+          { additionalTypenames: ['Update'] }
+        )
+        .toPromise()
+    );
+  },
+  async viewUpdateGroupsAsync({
+    limit,
+    offset,
+    appId,
+    filter,
+  }: ViewUpdateGroupsForAppQueryVariables): Promise<ViewUpdateGroupsForAppQuery> {
+    return await withErrorHandlingAsync(
+      graphqlClient
+        .query<ViewUpdateGroupsForAppQuery, ViewUpdateGroupsForAppQueryVariables>(
+          gql`
+            query ViewUpdateGroupsForApp(
+              $appId: String!
+              $limit: Int!
+              $offset: Int!
+              $filter: UpdatesFilter
+            ) {
+              app {
+                byId(appId: $appId) {
+                  id
+                  updateGroups(limit: $limit, offset: $offset, filter: $filter) {
+                    id
+                    group
+                    message
+                    createdAt
+                    runtimeVersion
+                    platform
+                    manifestFragment
```

Comment:
It doesn't look like this is used on this type, but if you want to keep querying it so that the shape of this data is the same as the other queries that's ok. Might be worth looking into a named fragment though for simplicity.

Reply:
👍 yeah, i mentioned above that i'm planning to follow this PR with one to replace update/updategroup schemas with fragments to keep things standardized.